### ISSUE TITLE
Panic when using openai

### ISSUE THREAD

**Issue by 4ydx** [dustinblackman/oatmeal#39]
I thought to try to submit a PR, but you have a git hook with cargo bin, which I'm not familiar with.  Anyway, here is a - probably incorrect - first attempt:

```
diff --git a/src/domain/services/bubble.rs b/src/domain/services/bubble.rs
index 9af582b..bc5f3e9 100644
--- a/src/domain/services/bubble.rs
+++ b/src/domain/services/bubble.rs
@@ -218,7 +218,7 @@ impl<'a> Bubble<'_> {
                 return line.len();
             })
             .max()
-            .unwrap();
+            .unwrap_or(0);

         if max_line_length > (self.window_max_width - line_border_width) {
             max_line_length = self.window_max_width - line_border_width;
```

After the change above is applied and compiled, I am seeing that "loading..." never goes away after chat gpt-4 completes sending data.  So there is something incompatible in the stream logic perhaps?

**Comment by dustinblackman** [comment]
Hey there! Thanks the report.

This sounds more like a bug in reading in the OpenAI responses rather than how bubbles are rendered. I'll take a look!

**Comment by 4ydx** [comment]
Agreed.  I simply wanted to stop the panic so that I could even begin to think about what might actually be going on.

**Comment by jim-at-jibba** [comment]
I assume I am getting the same error when using openai. Ollama models all work fine

**Comment by dustinblackman** [comment]
Sorry for the delay, fixed in [v0.12.4](https://github.com/dustinblackman/oatmeal/releases/tag/v0.12.4). Thanks!
